htcanalyze/main.py

```diff
@@ -44,18 +51,18 @@
                          "errors", "ram-history"]
 
 
-# class to store and change global variables
-class GlobalPlayer(object):
+# this class sets the output and input streams as global variables
+class GlobalStreams(object):
     """handle global variables."""
 
     def __init__(self):
-        """initialize."""
+        """ initialize class without redirecting streams as default """
```

Comment:
Why are we choosing to not redirect the streams?

This is a legitimate question. I've used the project, read some code, and opened a few PRs. I don't know why we don't want to redirect the streams here.

"as default" implies we might change this later in the script. Is that true? Or is the term "default" meant to say that we've changed the default to this?

Reply:
Either remove the docstring from `__init__` or from `class GlobalStreams`, because they are both functionally similar. The `__init__` after all is called automatically when instantiating the class. For automatic documentation tools, they are also considered equivalent.

You also don't need to mention that the `__init__` function initializes the class. And that it disables redirecting stdout is pretty obvious from the variable name. So it's better to just remove this docstring, and be more descriptive in the class docstring.